pilot/pkg/serviceregistry/serviceentry/servicediscovery.go

```diff
@@ -775,6 +775,11 @@ func (s *ServiceEntryStore) GetIstioServiceAccounts(svc *model.Service, ports []
 	return model.GetServiceAccounts(svc, ports, s)
 }
 
+func (s *ServiceEntryStore) NetworkGateways() map[string][]*model.Gateway {
+	// TODO implement mesh networks loading logic from kube controller if needed
```

Comment:
Does this mean the label will work only on k8s service, not SE? I think thats likely ok, just want to check

Reply:
Right - just shooting for parity with what we support today.